File: Java/JavaComplexity/app/src/main/java/com/dodemy/javacomplexity/dao/IFoodDAO.java
package com.dodemy.javacomplexity.dao;

import com.dodemy.javacomplexity.dto.FoodType;

import java.util.List;


import retrofit2.Call;
import retrofit2.http.GET;

public interface IFoodDAO {
    @GET("/discospiff/PlantPlacesKotlin/master/foodtypes.json")
    Call<List<FoodType>> getFoodTypes();
}

File: Kotlin/KotlinSimplicity/app/src/main/java/com/dodemy/kotlinsimplicity/dao/IFoodDAO.kt
package com.dodemy.kotlinsimplicity.dao

import com.dodemy.kotlinsimplicity.dto.FoodType
import retrofit2.http.GET

interface IFoodDAO {

    @GET("/discospiff/PlantPlacesKotlin/master/foodtypes.json")
    suspend fun getFoodTypes(): ArrayList<FoodType>
}